packages-ts/gauntlet-terra/src/lib/constants.ts

```diff
@@ -1 +1,2 @@
 export const ADDRESS_ZERO = 'terra000000000000000000000000000000000000000'
+export const LEDGER_ULUNA_PATH = "44'/330'/0'/0'/0"
```

Comment:
First, this should ref LUNA not ULUNA, but also it's not following the [BIP44 spec](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)., which defines the derivation path as:

```
m / purpose' / coin_type' / account' / change / address_index
```

Notice the extra `'` in your path declaration.

Additionally, the BIP44 is not Ledger specific in any way and can be used with a local private key also if we add HD capabilities to Gauntlet.

```suggestion
export const BIP44_LUNA_PATH = "44'/330'/0'/0/0"
```

Reply:
GH Issue (on Solana):

- https://github.com/smartcontractkit/chainlink-solana/issues/193